components/bl-leaflet-map/src/helpers/map.js

```diff
@@ -86,21 +86,49 @@ export function initMap(component, eventHandlers, map, currentLayer, uid) {
   });
 }
 
+function validateCircle(circle) {
+  const { point: { lat, lng }, radius, description } = circle;
+  const isAllPropertiesAreNumbers = [lat, lng, radius].every(value => {
+    const result = !isNaN(value);
+
+    if (!result) {
+      console.error(`Circle error!\n Expected number but received "${ value }" in\n`, circle);
+    }
+
+    return result;
+  });
+  const isDescriptionIsText = description === undefined || typeof description === 'string';
+  const hasRadius = radius > 0;
+
+  if (!isDescriptionIsText) {
+    console.error(`Circle Erorr!\n Expected description type text but received "${ description }" in\n`, circle);
+  }
+
+  if (isAllPropertiesAreNumbers && !hasRadius) {
+    console.error(`Circle error!\n Circle radius should be greater than 0, but received ${ radius } in\n`, circle);
+  }
+
+  return isAllPropertiesAreNumbers && isDescriptionIsText && hasRadius;
```

Comment:
we doesn't need all this variables just return false on the first validation error